bin/miden-cli/src/commands/new_transactions.rs

```diff
@@ -351,7 +351,7 @@ async fn execute_transaction(
     print_transaction_details(&transaction_execution_result)?;
     if !force {
         println!(
-            "\nContinue with proving and submission? Changes will be irreversible once the proof is finalized on the rollup (Y/N)"
+            "\nContinue with proving and submission? Changes will be irreversible once the proof is finalized on the rollup (y/N)"
```

Comment:
Let's take the chance to add this change as well:
```suggestion
            "\nContinue with proving and submission? Changes will be irreversible once the proof is finalized on the network (y/N)"
```

Reply:
Noted! Thanks for the feedback ignacio, changed here: https://github.com/0xPolygonMiden/miden-client/pull/843/commits/dd0033d2108a994b6dacadaf83aaa651164cb661